netbox/extras/scripts.py

```diff
@@ -446,6 +510,44 @@ def load_json(self, filename):
 
         return data
 
+    def run_test_scripts(self, job):
```

Comment:
Maybe `run_tests()` would be a better name for this? There's only one script involved (the class).